core/ErrorFlusherStdout.h

```diff
@@ -0,0 +1,31 @@
+#ifndef SORBET_ERROR_FLUSHER_STDOUT_H
+#define SORBET_ERROR_FLUSHER_STDOUT_H
+
+#include "core/AutocorrectSuggestion.h"
+#include "core/ErrorFlusher.h"
+#include "core/ErrorQueueMessage.h"
+#include <vector>
+
+namespace sorbet::core {
+
+class ErrorFlusherStdout : public ErrorFlusher {
+private:
+    std::vector<AutocorrectSuggestion> autocorrects;
+    bool printedAtLeastOneError{false};
+
+public:
+    ErrorFlusherStdout() = default;
+    ~ErrorFlusherStdout(){};
+    using ErrorFlusher::flushErrors;
```

Comment:
Why are these here? You shouldn't need these.

Reply:
I was getting a warning that I was hiding overloaded virtual function, and tried to find the answer online, and that's what I found